### DISCUSSION TITLE
Ability to sort Categories after creating them on left side

### DISCUSSION THREAD

**Discussion by superange128** [micah686/VnManager#22] [Ideas]
Currently you can create and add categories, but once they're created you can't sort them easily by dragging them around.

**Comment by micah686** [comment]
I didn't include that because I had the category search at top, where you can start typing, and it will filter the categories automatically. Is there a reason why you would prefer to have them sorted, instead of being able to search for categories?

**Reply by superange128** [reply]
It's more of a personal micromanage feature I like to do with list oriented stuff. I like customizing them to orders I want.

**Reply by micah686** [reply]
I'll consider it, but I would want a few more people requesting the feature before I try implementing it. I also think that while helpful, it's probably a lower priority change than some others.

**Reply by micah686** [reply]
Would just alphabetical work?

**Reply by superange128** [reply]
Alphabetical is a good start
